scenarios/reproducers/va-nfv-ovs-dpdk-sriov.yml

```diff
@@ -0,0 +1,152 @@
+# We can only use ansible_user_dir if you have the same user on the hypervisor
+# and locally - otherwise edit this
+cifmw_install_yamls_repo: "{{ ansible_user_dir }}/src/github.com/openstack-k8s-operators/install_yamls"
+# This will be created on the hypervisor.
+cifmw_basedir: "{{ ansible_user_dir }}/ci-framework-data"
+cifmw_path: "{{ ansible_user_dir }}/bin:{{ ansible_env.PATH }}"
+
+cifmw_reproducer_epel_pkgs:
+  - python3-bcrypt
+  - python3-passlib
+
+cifmw_libvirt_manager_pub_net: ocpbm
+
+# Automation section. Most of those parameters will be passed to the
+# controller-0 as-is and be consumed by the `deploy-va.sh` script.
+# Please note, all paths are on the controller-0, meaning managed by the
+# Framework. Please do not edit them!
+_arch_repo: "/home/zuul/src/github.com/openstack-k8s-operators/architecture"
+cifmw_architecture_scenario: "ovs-dpdk-sriov"
+
+# HERE if you want to overload kustomization, you can uncomment this parameter
+# and push the data structure you want to apply.
+# cifmw_architecture_user_kustomize:
+#   stage_0:
+#     'network-values':
+#       data:
+#         starwars: Obiwan
+
+# HERE, if you want to stop the deployment loop at any stage, you can uncomment
+# the following parameter and update the value to match the stage you want to
+# reach. Known stages are:
+#   pre_kustomize_stage_INDEX
+#   pre_apply_stage_INDEX
+#   post_apply_stage_INDEX
+#
+# cifmw_deploy_architecture_stopper:
+
+# What about some tempest?
+cifmw_run_tests: true
+cifmw_run_tempest: true
+cifmw_run_test_role: test_operator
+cifmw_test_operator_timeout: 7200
+cifmw_test_operator_tempest_include_list: |
+  tempest.scenario.test_network_basic_ops.TestNetworkBasicOps
+
+# This will instruct libvirt_manager to create
+# the ansible-controller, consuming the networks created and
+# managed by devscripts.
+# Note that the "osp_trunk" network is the equivalent of the
+# "private_net" in the CI layout, and will hold the VLAN for
+# network isolation.
+# The "ocpbm" network is managed by devscripts as well, and
+# provides access to Internet. This will be the equivalent of the
+# "public network" as seen in CI.
+cifmw_use_libvirt: true
+cifmw_virtualbmc_daemon_port: 50881
+cifmw_use_uefi: >-
+  {{ (cifmw_repo_setup_os_release is defined
+      and cifmw_repo_setup_os_release == 'rhel') | bool }}
+cifmw_root_partition_id: >-
+  {{
+    (cifmw_repo_setup_os_release is defined and cifmw_repo_setup_os_release == 'rhel') |
+    ternary(4, 1)
+  }}
+cifmw_libvirt_manager_net_prefix_add: false
+cifmw_libvirt_manager_fixed_networks:
+  - ocpbm
+  - ocppr
+  - osp_external
+  - osp_trunk
+cifmw_libvirt_manager_configuration:
+  networks:
+    ocpbm: |
+      <network>
+        <name>ocpbm</name>
+        <forward mode='bridge' />
+        <bridge name='ocpbm' />
+      </network>
+    ocppr: |
+      <network>
+        <name>ocppr</name>
+        <forward mode='bridge' />
+        <bridge name='ocppr' />
+      </network>
```

Comment:
afaik those two networks are managed by devscripts - not sure it's a good idea to redefine them here. @psathyan any thoughts?

Reply:
In this scenario, we don't allow `dev-scripts` to manage `ocpbm` and `ocppr` them. I would expect 
```
cifmw_devscripts_config_override:
  manage_br_bridge: 'n'
  manage_pro_bridge: 'n'
```

Why
- The mode is switched from `nat` to `bridge` as we need to add an external interface
- This interface / port that is being added to the has an associated VLAN id.

Reply:
I saw the downstream patch in the meantime - I'd expect to see those override here, in this file, even though they get overridden in downstream. Just for the sake of consistency :).

Reply:
IMHO it makes no sense to set `cifmw_devscripts_config_overrides` here because as you can see in DS patch the values provided there are very dependent on the specific setup (i..e it's not reproducible and it will always need to be overridden in DS).